knowledge-base/inputs-handle-keyboard-events.md

```diff
@@ -32,13 +32,15 @@ The keyboard events bubble up the DOM, so you can attach event handlers for them
 
 If you are looking for an event when the user confirms an action, consider the built-in `OnChange` event which fires when the user presses `Enter` or blurs the input.
 
+>important Optionally, you can disable the [**`DebounceDelay`**](https://docs.telerik.com/blazor-ui/components/textbox/overview#:~:text=DebounceDelay,int) parameter of the textbox by setting it to 0. The `DebounceDelay` default value is 150. It can fire the `onkeypress` event logic before the [**`ValueChanged`**](https://docs.telerik.com/blazor-ui/components/textbox/events#valuechanged) event of the input. 
```

Comment:
Use slugs instead of hard-coded URLs.

Parameters cannot be disabled - they are just set. Features can be disabled.